package.json

```diff
@@ -15,10 +15,10 @@
     "eslint-plugin-unicorn": "56.0.1"
   },
   "devDependencies": {
-    "@prettier/plugin-oxc": "0.1.2",
+    "@prettier/plugin-oxc": "0.1.3",
     "eslint": "8.57.1",
     "jest": "30.2.0",
-    "prettier": "3.7.3"
+    "prettier": "3.7.4"
```

Comment:
We can wait until this PR updates with `dependencies` changes, not just `devDependencies`.